File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string temp ="";
        for(int i=0;i<s.length();i++){
            if(isalnum(s[i])){
                temp+=tolower(s[i]);
            }
        }

        for(int i=0;i<temp.length();i++){
            if(temp[i] != temp[temp.length()-1-i])
            return false;
        }
        return true;
    }
};

File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s=s.replace(" ","")
        temp =""
        for i in range(len(s)):
            if(s[i].isalnum()):
               temp+=s[i].lower()

        rev = temp[::-1]
        if temp == rev:
            return True;
        else:
           return False;